src/master/embedded/embedding.md

```diff
@@ -1,10 +1,14 @@
-# Embedding immudb in your application
+# Key Value Store
 
-Using the Go client SDK means you are connecting to a immudb database server. There are cases where you don't want a separate server but embed immudb directly in the same application process, as a library.
+<WrappedSection>
+
+There are cases where you don't want a separate server but embed immudb directly in the same application process, as a library.
 
 immudb already provides an embeddable key-value store in the [embedded](https://github.com/codenotary/immudb/tree/master/embedded) package.
 The following example shows how to create or open a store, write some data and read it back.
 
```

Comment:
We should document what are the features missing from embedded KV store when compared to GRPC interface.